sktime/clustering/partitioning/_center_initializers.py

```diff
@@ -0,0 +1,161 @@
+# -*- coding: utf-8 -*-
+"""Cluster center initializers"""
+
+__author__ = ["Christopher Holder", "Tony Bagnall"]
+__all__ = ["ForgyCenterInitializer", "KMeansPlusPlusCenterInitializer"]
+
+import numpy as np
+from sklearn.utils import check_random_state
+
+from sktime.clustering.base.base import BaseClusterCenterInitializer
+from sktime.clustering.base._typing import (
+    NumpyArray,
+    NumpyRandomState,
+    CenterCalculatorFunc,
+)
+
+
+class ForgyCenterInitializer(BaseClusterCenterInitializer):
+    """Forgy Center Initializer used to create n centers
+    from a set of series using Forgys technique
+
+    Parameters
+    ----------
+    data_set: Numpy_Array
+        Numpy_Array that is the dataset to calculate the centers from
+
+    n_centers: int
+        Number of centers to be created
+
+    center_calculator_func: CenterCalculatorFunc
+        Function that is used to calculate new centers
+
+    random_state: NumpyRandomState, default = None
+        Generator used to initialise the centers.
+    """
+
+    def __init__(
+        self,
+        data_set: NumpyArray,
+        n_centers: int,
+        center_calculator_func: CenterCalculatorFunc = None,
+        random_state: NumpyRandomState = None,
+    ):
+        super(ForgyCenterInitializer, self).__init__(
+            data_set, n_centers, center_calculator_func, random_state
+        )
+
+    def initialize_centers(self) -> NumpyArray:
```

Comment:
```suggestion
    def _initialize_centers(self) -> NumpyArray:
```

Reply:
This must have got readded by mistake when i was fixing whatever happened with the pr the other day good spot

Reply:
This one actually needs to be public because I want to be able to call this from outside the class. So you construct the center initlizers and then when you want new centers you call initialize_centers()